public/index.html

```diff
@@ -116,109 +113,116 @@ <h3 class="pl-1 fontHeading">Create New Sprint</h3>
             </div>
         </div>
     </div>
+
     <!-- Dashboard---------------------------------------------------------------------------- -->
-    <section id="dashboard" class="bg-light">
+    <section class="bg-light" id="dashboard">
         <div class="container py-5">
-            <h3 class="py-3 fontHeading">Task Overview</h3>
+            <div class="row py-4">
+                <h3 class="fontHeading col-md-11 col-9" style="font-size: 24px;">Dashboard</h3>
+                <div class="col-md-1 col-3">
+                    <button type="submit" class="btn btn-primary" id="#" style="font-size: 14px;">Backlog</button>
```

Comment:
This button is not having id neither the implementation, please fix it.

Reply:
done